### DISCUSSION TITLE
Sunbird Developer Bootcamp | Kudos to Best Participants!

### DISCUSSION THREAD

**Discussion by pallakartheekreddy** [Sunbird-inQuiry/Community#69] [Announcements]
Hey folks! We would like to acknowledge and appreciate the best contributors among the participants who worked on Front-end problem statements in inQuiry, at Sunbird Developer Bootcamp.

Kudos to @Vinod-V3 and @snehar-nd for the best contributions!

Special mention to the participants who completed the Problem statement - 1 and raised the PR -  @panduankani @shreyasb22 @sajeshkayyath

Stay tuned for more exciting updates from the Community.

@pramodkvarma, @vrayulu @SanthoshVasabhaktula @mathewjpallan @surendrasinghs @PoojaJManjooran @sajeshkayyath @kameshbhr @vaibhavbhuva @rajnishdargan @krgauraw @Krishnaj20 @madhucr @rhwarrier @mmanimegalai @HarishGangula
@israr-trigyn @Xitija @vishalisakar @mansurskTarento @BharathTarento @adithyadinesh0412 @shashankpace @shwetha-rao13

**Comment by Krishnaj20** [comment]
Congrats @vinod-v3 @snehar-nd @panduankani @shreyasb22 @sajeshkayyath
